homeassistant/components/ekeybionyx/strings.json

```
@@ -0,0 +1,64 @@
+{
+  "config": {
+    "step": {
+      "pick_implementation": {
+        "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
+      },
+      "choose_system": {
+        "data": {
+          "system": "System"
+        },
+        "data_description": {
+          "system": "System the webhooks should be set up for"
+        },
+        "description": "Please select a system in which you want to set up the webhooks"
+      },
+      "webhooks": {
+        "description": "Please name your webhooks. Leaving them empty will delete them.",
+        "data": {
+          "webhook1": "Webhook 1",
+          "webhook2": "Webhook 2",
+          "webhook3": "Webhook 3",
+          "webhook4": "Webhook 4",
+          "webhook5": "Webhook 5",
+          "url": "URL"
+        },
+        "data_description": {
+          "webhook1": "Webhook 1",
+          "webhook2": "Webhook 2",
+          "webhook3": "Webhook 3",
+          "webhook4": "Webhook 4",
+          "webhook5": "Webhook 5",
+          "url": "URL"
+        }
+      },
+      "delete_webhooks": {
+        "description": "Webhook configuration inconsistent. Webhooks with Home Assistant were already configured. If you continue they will be marked for removal."
+      }
+    },
+    "error": {
+      "invalid_name": "Name is invalid",
+      "invalid_url": "URL is invalid",
+      "no_webhooks_provided": "No webhooks provided"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
+      "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
+      "oauth_error": "[%key:common::config_flow::abort::oauth2_error%]",
+      "oauth_failed": "[%key:common::config_flow::abort::oauth2_failed%]",
+      "oauth_timeout": "[%key:common::config_flow::abort::oauth2_timeout%]",
+      "oauth_unauthorized": "[%key:common::config_flow::abort::oauth2_unauthorized%]",
+      "missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]",
+      "authorize_url_timeout": "[%key:common::config_flow::abort::oauth2_authorize_url_timeout%]",
+      "no_url_available": "[%key:common::config_flow::abort::oauth2_no_url_available%]",
+      "user_rejected_authorize": "[%key:common::config_flow::abort::oauth2_user_rejected_authorize%]",
+      "no_availible_webhooks": "There are no availible webhooks in the ekey Bionyx plattform. Please delete some and try again.",
```

Typo in both the key and the string:

```suggestion
      "no_available_webhooks": "There are no available webhooks in the ekey Bionyx plattform. Please delete some and try again.",
```